mfc.sh

```diff
@@ -94,7 +94,6 @@ if [ "$1" == 'load' ]; then
     # Reset modules to default system configuration
     if [ "$u_c" != 'p' ]; then
         module reset > /dev/null 2>&1
-        code="$?"
```

Comment:
Did you intend to include this change?

Reply:
@henryleberre, same here. There should be no (intended) changes to this file other than the ones I followed from you

Reply:
remove this change

Reply:
These were technically added by one of your commits (authored and committed you) although some of the changes appear to may have been some of mine. I'm not sure why my commits would disappear and be squashed without a co-author attribution. In any case, you can remove these changes.